### ISSUE TITLE
[Discussion] add racket-minimal and move racket to extras bucket

### ISSUE THREAD

**Issue by batkiz** [ScoopInstaller/Main#693]
I am a racket user, and I almost only use the cli Racket application. So the racket app in this bucket is a bit large for me. 

These days I found there is a racket-minimal provided in offical website which only provide basic cli apps, with no GUI application known as DrRacket.

Also, as listed in [Criteria for including apps in the main bucket](https://github.com/lukesampson/scoop/wiki/Criteria-for-including-apps-in-the-main-bucket), there should be no GUI tool in the main bucket. So I want to add racket-minimal to the main bucket and move the standard racket app(with GUI) to extras bucket, I hope the miantainers could help me with this work.

---

I've [added racket-minimal to my personal bucket](https://github.com/batkiz/backit/blob/master/bucket/racket-minimal.json) and tested it. It works perfectly for me but I'm not confident to the mainfest especially the `autoupdate` part. Would anybody please help me test it?

**Comment by Ash258** [comment]
**TL;DR** Add minimal version into Main bucket

There is no reason for migration when you can just add minimal version in Main bucket.

> Also, as listed in Criteria for including apps in the main bucket, there should be no GUI tool in the main bucket.

This is for graphical applications **only** (eg. Which cannot be fully used from command line. Vscode as example, it has command line support, but you cannot edit files from command line).
Main bucket purpose is for tools, which primary usage is from command line. If they support GUI it is not problem., Powersherll, git, 7zip, Lessmsi, ... for example

**Comment by batkiz** [comment]
Sorry for my late reply.
I'll improve the PRs.
